analyzer/codechecker_analyzer/analyzers/cppcheck/analyzer.py

```diff
@@ -0,0 +1,350 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+"""
+"""
+from distutils.version import StrictVersion
```

Comment:
I'll mark it as a TODO, but we depend on `distutils` lot of places, I would prefer to create an issue -> do it in a separate pr if you don't mind.

Reply:
(It's strange that they dropped the versioning feature as a whole, but alright, it could be a separate issue. Ubuntu 20.04 comes with Python 3.8, 22.04 with 3.10.)